orttraining/orttraining/python/training/ortmodule/_utils.py

```diff
@@ -38,33 +38,33 @@ def _ortvalue_from_torch_tensor(torch_tensor):
     return C.OrtValue.from_dlpack(to_dlpack(torch_tensor), is_bool_tensor)
 
 
-def _torch_tensor_from_dl_pack(dlpack, ortvalue, device):
-    torch_tensor = from_dlpack(dlpack) if device.type != 'ort' else C.ort_from_dlpack(dlpack)
+def _torch_tensor_from_dl_pack(dlpack, ortvalue):
+    torch_tensor = from_dlpack(dlpack)
     return torch_tensor.to(torch.bool) if ortvalue.data_type() == 'tensor(bool)' else torch_tensor
 
 
 def _ortvalue_to_torch_tensor(ortvalue, device):
+    if device.type == 'ort':
```

Comment:
Is `at:Tensor` the same as `torch.Tensor`? If not, we have the same multi-type return issue here

Reply:
the same